opensearch-operator/controllers/cluster_test.go

```diff
@@ -186,11 +186,15 @@ var _ = Describe("Cluster Reconciler", func() {
 			sts := &appsv1.StatefulSet{}
 			Eventually(func() error {
 				return k8sClient.Get(context.Background(), types.NamespacedName{
-					Name:      fmt.Sprintf("%s-client", OpensearchCluster.Name),
+					Name:      fmt.Sprintf("%s-master", OpensearchCluster.Name),
 					Namespace: OpensearchCluster.Namespace,
 				}, sts)
 			}, timeout, interval).Should(Succeed())
-			Expect(sts.ObjectMeta.Labels).To(HaveKeyWithValue("quux", "quut"))
+			Expect(sts.Spec.Template.Spec.TopologySpreadConstraints[0].TopologyKey).To(Equal("zone"))
```

Comment:
Please move this to the correct `It` block. Even that means repeating the get for the sts, but it makes it clear what is tested. Any reason why you deleted the `Expect(sts.ObjectMeta.Labels).To(HaveKeyWithValue("quux", "quut"))` line?

Reply:
Oops, I thought it was github showing a weird change, but it was me. Restored the original test and moved the test to the correct `It` block.